Formula/step.rb

```diff
@@ -0,0 +1,27 @@
+class Step < Formula
+  desc "Crypto and x509 Swiss-Army-Knife"
+  homepage "https://github.com/smallstep/cli"
```

Comment:
Would it make sense to change the homepage to https://smallstep.com since the formula name is simply `step` and it provides two cli utilities from two separate git repositories?